pywbem_mock/_wbemconnection_mock.py

```diff
@@ -626,14 +714,20 @@ def compile_dmtf_schema(self, schema_version, schema_root_dir, class_names,
           TypeError: The 'schema_version' is not a valid tuple with 3
             integer components
           :exc:`~pywbem.MOFCompileError`: Compile error in the MOF.
-        """
+        """  # noqa: E501
+        # pylint: enable:line-too-long
+
+        warnings.warn(
+            "Calling compile_dmtf_schema is deprecated; it will be removed "
+            "in the next pywbem release after 1.0.0. Use",
+            DeprecationWarning, 2)
 
         schema = DMTFCIMSchema(schema_version, schema_root_dir,
                                use_experimental=use_experimental,
                                verbose=verbose)
-        schema_mof = schema.build_schema_mof(class_names)
+        schema_pragma_mof = schema.build_schema_mof(class_names)
```

Comment:
It is just a local variable, but still worth thinking about naming consistency. Originally I thought it should be renamed to `schema_mof_pragma` for consistency with the various occurrences of `schema_mof_pragma_file`, but that is somehow not logical for a MOF string.

Which lead me to think that `schema_mof_pragma_file` is probably unlogical, too, at least when comparing it with `schema_mof_pragma`.

So how about renaming the various occurrences of `schema_mof_pragma_file` to either `schema_pragma_mof_file` or somewhat shorter to `schema_pragma_file`?